src/Sulu/Component/Content/Compat/Structure/LegacyPropertyFactory.php

```diff
@@ -68,7 +68,8 @@ public function createTranslatedProperty($property, $locale, StructureInterface
     /**
      * Create a new property.
      *
-     * @param Item $item
+     * @param ItemMetadata $property
+     * @param null|StructureInterface $structure
```

Comment:
Actually we were saying that we only add these params if they add some value. In that case all this information is already in the function signature itself, therefore I would just remove these two lines.